docs/platforms/react-native/index.mdx

```diff
@@ -80,6 +81,10 @@ Sentry.init({
   // profilesSampleRate is relative to tracesSampleRate.
   // Here, we'll capture profiles for 100% of transactions.
   profilesSampleRate: 1.0,
+  // Record Session Replays for 10% of Sessions and 100% of Errors
```

Comment:
RN will have the `enableExperimentalViewRenderer` option soon ([PR in review](https://github.com/getsentry/sentry-react-native/pull/4660)), same considerations as in https://github.com/getsentry/sentry-docs/pull/13034/files#r2001573132 apply